### PR TITLE
Sync with API 1.9.5

### PR DESCRIPTION
**Pull request by hudokkow**

### CODE REVIEW

**Comment by bkuhls** [comment]
wfm

**Comment by hudokkow** [comment]
Great, thanks!

File: src/client.cpp [hunk 1]
```diff
@@ -65,9 +66,9 @@ CStdString g_strUserPath             = "";
 CStdString g_strClientPath           = "";
 CStdString		g_AddonDataCustom	= "";					// location of custom addondata settings file
 
-CHelper_libXBMC_addon *XBMC           = NULL;
-CHelper_libXBMC_pvr   *PVR            = NULL;
-CHelper_libXBMC_gui   *GUI            = NULL; 
+CHelper_libXBMC_addon   *XBMC           = NULL;
+CHelper_libXBMC_pvr   	*PVR            = NULL;
+CHelper_libKODI_guilib  *GUI            = NULL;
```
Comment (AlwinEsch) [inline src/client.cpp]: Can you hold the spaces in the right flow.

**Comment by hudokkow** [comment]
These files are a mess with spaces and tabs mixed all over the place.
imo, correcting the indent mess is work for another PR.

**Comment by AlwinEsch** [comment]
OK, no problem. Yeah I know there is always a big muddled up, from KODI side must be everything in his style and changes already with this.

**Pull request merged**